README.md

```diff
@@ -17,6 +17,8 @@ This library targets .NET 4.5.2 and can be installed via Nuget :
 Install-Package InterFAX.Api -Version 1.0.5
 ```
 
+.NET 3.5/4.0 solutions must make calls directly to the REST API [(documentation)](https://www.interfax.net/en/dev/rest/reference) [(sample code)](https://github.com/kfwls/interfax-dotnet).
```

Comment:
@kfwls thanks for adding this, but we've put this in the samples repo. Could you update it? Then I'm happy to merge.

https://github.com/interfax/interfax-dotnet-samples/blob/master/extra_5_send_individual_fax_dotnet_35.cs